lib/rbslicer.rb

```diff
@@ -57,19 +57,29 @@
   #    ...  "field_name" => "ruby-string-field"
   #    ....}
 class SlicingDice < Rbslicer::SlicingDiceAPI
-  def initialize(
-      master_key: nil, custom_key: nil, read_key: nil,
-          write_key: nil, timeout: 60, use_ssl: true, uses_test_endpoint: false)
+  def initialize(options = {})
+      master_key= options.fetch(:master_key, nil)
+      custom_key = options.fetch(:custom_key, nil)
+      read_key = options.fetch(:read_key, nil)
+      write_key = options.fetch(:write_key, nil)
+      timeout = options.fetch(:timeout, 60)
+      use_ssl = options.fetch(:use_ssl, true)
```

Comment:
I think use_ssl is unused now, as you changed the behavior from requester.rb. Remove all the references for this variable.